README.md

```diff
@@ -8,3 +8,9 @@
 ## Contributing
 
 Contributions are always welcomed! Please see [CONTRIBUTING](./CONTRIBUTING.md) for detailed guidelines.
+Sentinel Golang
+
+## core package
```

Comment:
We may put this to the wiki instead of here :)

Reply:
ok